resource/js/tab-hierarchy.js

```diff
@@ -161,13 +161,35 @@ const tabHierApp = Vue.createApp({
         width: width + 'px'
       }
     },
-    compareLabels (a, b) {
-      // Set labels as label, prefLabel or an empty string if there are no lables
-      const labelA = a.label || a.prefLabel || ''
-      const labelB = b.label || b.prefLabel || ''
-      const lang = window.SKOSMOS.content_lang ? window.SKOSMOS.content_lang : window.SKOSMOS.lang
+    compareStrs (a, b) {
```

Comment:
I see you've renamed the function from `compareLabels` to `compareStrs`, to reflect that it's no longer comparing just labels. However, since it takes two concepts (`a` and `b`) as parameters, I'd suggest calling it `compareConcepts` instead.